fdm-app/app/routes/farm.$b_id_farm.settings.properties.tsx

```diff
@@ -0,0 +1,216 @@
+                                            </FormControl>
+                                            <FormDescription />
+                                            <FormMessage />
+                                        </FormItem>
+                                    )}
+                                />
+                            </div>
+                            <div className="flex flex-col space-y-1.5 col-span-1">
+                                <FormField
+                                    control={form.control}
+                                    name="b_postalcode_farm"
+                                    render={({ field }) => (
+                                        <FormItem>
+                                            <FormLabel>Postcode</FormLabel>
+                                            <FormControl>
+                                                <Input placeholder="bv. 1234 AB" {...field} autoComplete="postal-code" maxLength={300} />
+                                            </FormControl>
+                                            <FormDescription />
+                                            <FormMessage />
+                                        </FormItem>
+                                    )}
+                                />
+                            </div>
+                        </div >
+                    </fieldset>
+                    <br />
+                    <div className="ml-auto">
+                        <Button
+                            type="submit"
+                            disabled={form.formState.isSubmitting}
+                            className="m-auto"
+                        >
+                            {form.formState.isSubmitting && <LoadingSpinner />}
+                            Bijwerken
+                        </Button>
+                    </div>
+
+                </Form>
+            </RemixFormProvider>
+        </div>
+    )
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+    const b_id_farm = params.b_id_farm;
+
+    if (!b_id_farm) {
+        return dataWithError(null, "Missing farm ID.");
+    }
+
+    try {
+        const formValues = await extractFormValuesFromRequest(request, FormSchema);
+
+        await updateFarm(
+            fdm,
+            b_id_farm,
+            formValues.b_name_farm,
+            formValues.b_businessid_farm,
+            formValues.b_address_farm,
+            formValues.b_postalcode_farm
+        );
+
+        return dataWithSuccess(
+            `farm is updated`,
+            { message: `${formValues.b_name_farm} is bijgewerkt! 🎉` }
+        );
+    } catch (error) {
+        console.error("Failed to update farm:", error);
+        return dataWithError(
+            null,
+            `Er is iets misgegaan bij het bijwerken van de bedriijfgegevens: ${error instanceof Error ? error.message : 'Onbekende fout'}`
+        );
```

Comment:
replace with bedrijfsgegevens. spelling.

Reply:
Resolved in 300057d2acb3d18ad9179b4d43632bf525abb315